src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx

```diff
@@ -112,27 +120,35 @@ const BottomSheetBackdropComponent = ({
     },
     [disappearsOnIndex]
   );
-  //#endregion
 
-  return pressBehavior !== 'none' ? (
-    <TapGestureHandler onGestureEvent={gestureHandler}>
+  const AnimatedView = () => {
+    return (
       <Animated.View
         style={containerStyle}
         pointerEvents={pointerEvents}
-        accessible={true}
-        accessibilityRole="button"
-        accessibilityLabel="Bottom Sheet backdrop"
-        accessibilityHint={`Tap to ${
-          typeof pressBehavior === 'string' ? pressBehavior : 'move'
-        } the Bottom Sheet`}
+        accessible={_providedAccessible ?? undefined}
```

Comment:
Why would you mark the root `View` as `accessible` for the bottom sheet? That makes it so you cannot focus any of the descendant views (which is all of the sheet content) with a screen reader.